handlers/main.yml

```diff
@@ -78,3 +78,6 @@
 
 - name: reset audit pkg ugids
   command: rpm --setugids audit
+
+- name: remount /tmp
```

Comment:
It doesn't. I can't use it for modifying /etc/fstab because I don't know which block device /tmp is mounted from, and if I use it in the handler like this

```
- name: remount /tmp
  mount:
      name: /tmp
      opts: remount
      state: mounted
```

the /tmp filesystem is not remounted, i.e. running "mount" still shows the old mount options.